Step 1: Motif finding

Step live from: Monday April 22nd noon Eagle Time (1 pm CET/noon BST/7 am ET/4 am PT)

Motif identification is one of the oldest tasks to which bioinformaticians set their sights, back in the dark ages of the field, when computational caveman punched holes into stone cards. And, yet, motif finding continues to be a key component of the bioinformatics toolset; if anything, it has become even more important in the genomics-era, where patterns in the data can help us make sense of overwhelming volumes of base pairs. As such, motif finding notably played a major role in the ENCODE project, an undertaking to functionally annotate the human genome that was published last year.

The challenge

In this, our opening challenge, the task is to use k­-mer counting in order to identify the most overrepresented 7bp motif in a set of sequences (forward strand only). As this is Step 1, we have made it quite easy: the ultra-tricky stuff is still to come!


Once you have your solution, you can use it to determine the URL of Step 2, as follows: http://genomebiology.com/about/update/DNA60_MOTIF

For example, if you find the most overrepresented motif to be GATTACA, then Step 2’s URL will be: http://genomebiology.com/about/update/DNA60_GATTACA


Archived from: http://genomebiology.com/about/update/DNA60_STEPONE

