Step 4: Metagenomics

Step live from: Wednesday April 24th 5 pm Eagle Time (6 pm CET/5 pm BST/noon ET/9 am PT)

The metagenome, where a sample contains DNA derived from multiple, co-habiting species, was not an immediate beneficiary of the next-generation sequencing revolution. The longer reads needed to confer phylogenetic assignments without ambiguity exceeded the specs of the cheapest platforms. In recent years, all this has changed, and all manner of interesting questions are now being answered from metagenomic data – such as the effect of formula milk on the infant gut microbiome, the impact of fish tanks on the indoor microbiome and how much of the DNA in your poop originated in your food.

Alongside the wealth of new data, many novel methods have been developed to analyze the metagenome – many of which rely on 16S/18S as a phylogenetic marker. Examples published by Genome Biology include MetAMOS, Ray Meta, EMIRGE and LEfSe.



The challenge

For the Step 4 challenge, the task is to find the genus with the highest abundance in a simulated 16S metagenomic dataset.

The answer will enable you to form the URL for tomorrow's final! challenge. For example, if the highest abundance is the genus Halomonas (as if we would be cheeky enough to use a sample originating from an #arseniclife study!), then the URL for tomorrow's challenge would be: http://genomebiology.com/about/update/DNA60_HALOMONAS

Archived from: http://genomebiology.com/about/update/DNA60_CARB
